tests/scripts/create-dev-cluster.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+#set -eEx
+KUBECTL="minikube kubectl --"
+ROOK_EXAMPLES_DIR="../../deploy/examples/"
+
+wait_for_ceph_cluster() {
+    echo "Waiting for ceph cluster"
+    WAIT_CEPH_CLUSTER_RUNNING=20
+    while ! $KUBECTL get cephclusters.ceph.rook.io -n rook-ceph -o jsonpath='{.items[?(@.kind == "CephCluster")].status.ceph.health}' | grep -q "HEALTH_OK"; do
+	echo "Waiting for Ceph cluster installed"
+	sleep ${WAIT_CEPH_CLUSTER_RUNNING}
+    done
+    echo "Ceph cluster installed and running"
+}
+
+get_minikube_driver() {
+    os=$(uname)
+    architecture=$(uname -m)
+    if [[ "$os" == "Darwin" ]]; then
+        if [[ "$architecture" == "x86_64" ]]; then
+            echo "hyperkit"
+        elif [[ "$architecture" == "arm64" ]]; then
+            echo "qemu"
+        else
+            echo "Unknown Architecture on Apple OS"
+	    exit 1
+        fi
+    elif [[ "$os" == "Linux" ]]; then
+        echo "kvm2"
+    else
+        echo "Unknown/Unsupported OS"
+	exit 1
+    fi
+}
+
+show_ceph_dashboard_info() {
+    DASHBOARD_PASSWORD=$($KUBECTL -n rook-ceph get secret rook-ceph-dashboard-password -o jsonpath="{['data']['password']}" | base64 --decode && echo)
+    IP_ADDR=$($KUBECTL get po --selector="app=rook-ceph-mgr" -n rook-ceph --output jsonpath='{.items[*].status.hostIP}')
+    PORT="$($KUBECTL -n rook-ceph -o=jsonpath='{.spec.ports[?(@.name == "dashboard")].nodePort}' get services rook-ceph-mgr-dashboard-external-http)"
+    BASE_URL="http://$IP_ADDR:$PORT"
+    echo "==========================="
+    echo "Ceph Dashboard: "
+    echo "   IP_ADDRESS: $BASE_URL"
+    echo "   PASSWORD: $DASHBOARD_PASSWORD"
+    echo "==========================="
+}
+
+setup_minikube_env() {
+    minikube_driver="$(get_minikube_driver)"
+    echo "Setting up minikube env (using $minikube_driver driver)"
+    minikube delete
+    minikube start --disk-size=40g --extra-disks=1 --driver "$minikube_driver"
+    eval "$(minikube docker-env -p minikube)"
+}
+
+create_rook_cluster() {
+    echo "Creating cluster"
+    $KUBECTL apply -f crds.yaml -f common.yaml -f operator.yaml
+    $KUBECTL apply -f cluster-test.yaml -f toolbox.yaml
+    $KUBECTL apply -f dashboard-external-http.yaml
+}
+
+check_examples_dir() {
+    CRDS_FILE="crds.yaml"
+    if [ ! -e ${CRDS_FILE} ]; then
+	echo "File ${ROOK_EXAMPLES_DIR}/${CRDS_FILE} does not exist. Please, provide a valid rook examples directory."
+	exit 1
+    fi
+}
+
+wait_for_rook_operator() {
+    echo "Waiting for rook operator"
+    $KUBECTL rollout status deployment rook-ceph-operator -n rook-ceph --timeout=180s
+    while ! $KUBECTL get cephclusters.ceph.rook.io -n rook-ceph -o jsonpath='{.items[?(@.kind == "CephCluster")].status.phase}' | grep -q "Ready"; do
+	echo "Waiting for cluster to be ready..."
+	sleep 20
+    done
+}
+
+enable_rook_orchestrator() {
+    echo "Enabling rook orchestrator"
```

Comment:
When will the rook orchestrator be ready to enable by default?

Reply:
@travisn I think once we fix all the issues we are working on we should be in a good spot to enable it by default.